prestoadmin/util/presto_config.py

```diff
@@ -72,7 +73,7 @@ def coordinator_config():
     def _lookup(self, key):
         result = self.config_properties.get(key, self.default_config[key])
         if not result:
-            raise Exception(
+            error(
```

Comment:
are there any other places where you raise an Exception rather than doing error?